CriteoPublisherSdk/Tests/UnitTests/Network/CR_ApiHandlerTests.m

```diff
@@ -94,6 +94,7 @@ - (void)testCallCdb {
                  config:self.configMock
              deviceInfo:self.deviceInfoMock
                 context:self.contextData
+                  coppa:nil
```

Comment:
Please could you add a test case checking that this object is actually transmitting the flag

Reply:
I have added a test called `testCallCdbWithCoppa` that checks the transmission of the flag.
Resolving this conversation.